src/main/scala/coreplex/BaseTile.scala

```diff
@@ -0,0 +1,56 @@
+// See LICENSE.SiFive for license details.
+
+package coreplex
+
+import Chisel._
+import config._
+import diplomacy._
+import rocket.{TileInterrupts, XLen}
+import uncore.tilelink2._
+import util.GenericParameterizedBundle
+
+abstract class BareTile(implicit p: Parameters) extends LazyModule
+
+abstract class BareTileBundle[+L <: BareTile](_outer: L) extends GenericParameterizedBundle(_outer) {
+  val outer = _outer
+  implicit val p = outer.p
+}
+
+abstract class BareTileModule[+L <: BareTile, +B <: BareTileBundle[L]](_outer: L, _io: () => B) extends LazyModuleImp(_outer) {
+  val outer = _outer
+  val io = _io ()
+}
+
+// Uses a tile-internal crossbar to provide a single TileLink master port
+trait TileNetwork {
+  implicit val p: Parameters
+  val module: TileNetworkModule
+  val l1backend = LazyModule(new TLXbar)
+  val masterNodes = List(TLOutputNode())
+  masterNodes.head := l1backend.node
+}
+
+trait TileNetworkBundle {
+  val outer: TileNetwork
+  val master = outer.masterNodes.head.bundleOut
```

Comment:
As discussed offline, either get Chisel PR #366 merged, or use an OutputNode for multiple connections.